src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -2938,6 +3189,10 @@ private Debugger PopActiveDebugger()
                             _steppingMode = SteppingMode.StepIn;
                             _overOrOutFrame = _nestedRunningFrame;
                             _nestedRunningFrame = null;
+                            if (_lastActiveDebuggerAction == DebuggerResumeAction.StepOut)
+                            {
+                                CancelDebuggerProcessingAsNeeded(poppedDebugger);
```

Comment:
Thinking about this some more I have changed my mind.  Right now "step-out" aborts the Invoke-Command debugging and continues with local script debugging, which I thought was the right behavior for some reason.  But this is different behavior from how "continue" and "quit" debugger commands work.  I think "step-out" should be consistent and I'll remove this cancel code.